### PR TITLE
XDR 3.0 add api changes support

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
This PR supports only the following two changes from the issue:

- image

- image

related issue: https://github.com/demisto/etc/issues/39900

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CortexXDR/IncidentFields/XDR_Status_v2.json [hunk 1]
```diff
@@ -28,7 +28,8 @@
 	"selectValues": [
         "new",
         "under_investigation",
-		"resolved_threat_handled",
+		"resolved_true_positive",
+		"resolved_security_testing",
```
Comment (dansterenson) [inline Packs/CortexXDR/IncidentFields/XDR_Status_v2.json:32]: [in page 38 ](https://docs.paloaltonetworks.com/content/dam/techdocs/en_US/pdf/cortex/cortex-xdr/cortex-xdr-api/cortex-xdr-api.pdf) you can see that the RESOLVED_THREAT_HANDLED was replaced with RESOLVED_TRUE_POSITIVE and RESOLVED_SECURITY_TESTING was added

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 2]
```diff
@@ -452,27 +454,31 @@ def get_endpoints(self,
             endpoints = reply.get('reply').get('endpoints', [])
         return endpoints
 
-    def isolate_endpoint(self, endpoint_id):
+    def isolate_endpoint(self, endpoint_id, incident_id=None):
+        request_data = {
+            'endpoint_id': endpoint_id
```
Comment (yaakovi) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:459]: [PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
everywhere else too
```suggestion
            'endpoint_id': endpoint_id,
```

File: Packs/CortexXDR/ReleaseNotes/4_0_2.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+#### Incident Fields
+- **XDR Status v2**
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Maintenance and stability enhancements.
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply Packs/CortexXDR/ReleaseNotes/4_0_2.md:7]: Done

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py (hunk 2 above)
Comment (dansterenson) [reply Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:459]: Done

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py [hunk 4]
```diff
@@ -2643,7 +2665,8 @@ def test_run_script_kill_multiple_processes_command(requests_mock):
     args = {
         'endpoint_ids': endpoint_ids,
         'timeout': timeout,
-        'process_name': processes_names
+        'process_name': processes_names,
+        'incident_id': '4',
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py:2669]: Done

File: Packs/CortexXDR/IncidentFields/XDR_Status_v2.json (hunk 1 above)
Comment (dansterenson) [reply Packs/CortexXDR/IncidentFields/XDR_Status_v2.json:32]: stays with the original value

**Review by yaakovi — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review completed.

**Comment by dansterenson** [comment]
@yaakovi I returned the resolved_threat_handled field since @Itay4 asked not to remove it for now as there is a customer that did not move to 3.0 just yet and it will break for him.
We are waiting for the customer response but apparently this pr will also fix the issue ([here](https://github.com/demisto/etc/issues/41457).

**Review by yaakovi — APPROVED** [review]

**Pull request merged**